custom_components/powercalc/data/tp-link/L530/color_temp.csv

```diff
@@ -0,0 +1,1353 @@
+bri,mired,watt
+1,153,0.589
+1,163,0.589
+1,173,0.589
+1,183,0.589
+1,193,0.589
+1,203,0.589
+1,213,0.589
+1,223,0.589
+1,233,0.589
+1,243,0.589
+1,253,0.589
+1,263,0.589
+1,273,0.589
+1,283,0.589
+1,293,0.589
+1,303,0.589
+1,313,0.589
+1,323,0.589
+1,333,0.589
+1,343,0.589
+1,353,0.589
+1,363,0.589
+1,373,0.589
+1,383,80.589
```

Comment:
Typo here.
And make sure to gzip the corrected color_temp.csv and hs.csv.
And remove the non gz files from the PR.